src/app.rs

```diff
@@ -181,6 +181,10 @@ pub struct Cli {
     #[arg(long, hide = !cfg!(windows))]
     pub system_protected: bool,
 
+    /// When showing files do not quote their names
+    #[arg(long)]
+    pub literal: bool,
```

Comment:
`-N` and description align with https://man7.org/linux/man-pages/man1/ls.1.html

Reply:
Fixed it!